### ISSUE TITLE
Do some vendors require large file uploads?

### ISSUE THREAD

**Issue by uniphil** [uniphil/hes-vendor-dash-issues#9]
> 9. Vendor #68 (I deleted this vendor’s account yesterday and they re-registered) says: Is there a limit to how big the pdf can be? Mine vary from 200 to 100 mb. That being said, my jpeg example pages are under 200kb and those are not uploading as well.
> 
> [phil note: if huge (>10MB) files are required, this will likely require new feature development work]

**Comment by SuperSparkplug** [comment]
Some Vendors might, as they'll be selling digital books of various sizes, but if that's the case and it's not possible to increase the 10MB limit, perhaps it might be better to have some sort of note saying to contact an e-mail address about it.

Let me know your thoughts.

**Comment by uniphil** [comment]
Yeah, some kind of work around like that might be easiest, but I'm worried that this might be something that a lot of vendors need.

I moved it back to triage because I think we should loop more folks into the discussion. There are ways we can handle on the back-end if it's important enough to be worth the time.

**Comment by uniphil** [comment]
Quick update on this:

I finally got around to checking the server configuration, and it's set up for a **100MB limit**. The 88.6MB file I tested with worked fine.

Currently a vendor is asking Cali about a 200MB file. Not sure the best way to handle that with them, but as @SuperSparkplug suggested, we might get away with just special-casing exceptional >100MB cases and work out some kind of solution to get the files from them directly.
